### ISSUE TITLE
Multiple authors aren't displayed

### ISSUE THREAD

**Issue by USLTD** [tfonteyn/NeverTooManyBooks#6]
Hello,
I have added custom book (which isn't in any of library databases) and book has 3 authors. I have tried to add all 3 but it only display first author not others. Is this intended action or just a bug?

**Comment by tfonteyn** [comment]
Hi,

It depends where you're looking.
In detail view (and obviously when editing) of the book, all three should show as a list.

First (you may already have found this) if you edit a book, on the "details" tab you have the author field with a pen-icon.
Fields with pen-icons open a new edit screen when tapping them.

For the author, you will get a text box at the top; enter a name and tap the "+" or the 'other' "+" (with details) button in the taskbar.
The author will be added to the list underneath the textbox. use the "2-bars" button to re-order authors if needed. Top one is considered "primary/main". When going back a screen, all 3 will show up in the fields (as a 1-line list)

Secondly:
In the main book list, by default a book is only shown under the primary author (i.e. the first one in the list) but you can change that when editing the styles.

Here's an example: Open the menu panel and select "Booklist styles" (read the 'tip').
If you haven't done this before, the top one is the default "Author, Series". Either long-click or use the menu button on the row (3-dots) and choose 'duplicate'.
Modify the name to something that suits you.
Read the tip :)

Plenty of things to customize, but you're looking for ... scroll down... "Show books with multiple authors"
Switch that ON and you get "under each owner".

In the book list you will now see all 3 authors on individual rows, and that single book will appear once under each.

There is A LOT you can customize using styles.

Hope this helps. Feel free to ask more help. I'm trying to add to the docs based on incoming questions :)

PS: when on the book-details (or edit) you can long-click the placeholder for the cover, and use the camera to make a picture of your book.

**Comment by USLTD** [comment]
Sorry for the late reply but I meant the fact that when I clicked on a multi-author book, it only displayed the first author entry, not others. Looks like it only affects books with non-English author names, causes if I translated author names into English, it would display all of them.

**Comment by USLTD** [comment]
FYI, the language I used originally was Georgian.

**Comment by tfonteyn** [comment]
no worry - happy to help.

Can you give me the details of that book  with full Georgian spelling. I can then try it out.

edit, no worry. I just found a list [https://ka.wikipedia.org/wiki/%E1%83%A5%E1%83%90%E1%83%A0%E1%83%97%E1%83%95%E1%83%94%E1%83%9A%E1%83%98_%E1%83%9B%E1%83%AC%E1%83%94%E1%83%A0%E1%83%9A%E1%83%94%E1%83%91%E1%83%98%E1%83%A1_%E1%83%A1%E1%83%98%E1%83%90](https://ka.wikipedia.org/wiki/%E1%83%A5%E1%83%90%E1%83%A0%E1%83%97%E1%83%95%E1%83%94%E1%83%9A%E1%83%98_%E1%83%9B%E1%83%AC%E1%83%94%E1%83%A0%E1%83%9A%E1%83%94%E1%83%91%E1%83%98%E1%83%A1_%E1%83%A1%E1%83%98%E1%83%90)

**Comment by tfonteyn** [comment]
indeed - I can reproduce this easily. I'll need to investigate and then get back to you.

**Comment by tfonteyn** [comment]
Hi,

I've found the issue - it seems I was to strict in applying ascii rules even on non-latin scripts such as Georgian (and Greek etc) when it comes to sorting and comparing. 

I've added a partial fix for this issue. which will be included in a new 4.3 release later this week.
This should fix the compare-issue, and hence fix your list of authors problem fully.

Sorting **might** not be perfect - my knowledge of non-latin script is sadly non-existent, so I cannot be sure.
I will certainly look further into this for a future update.

I'll add a note here when 4.3 is out.
